nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -64,6 +64,12 @@
           this version for the entire lifecycle of the 21.11 release.
         </para>
       </listitem>
+      <listitem>
+        <para>
+          <literal>lispPackages.nyxt</literal> has been renamed to
+          <literal>lispPackages.nyxt-unwrapped</literal>
+        </para>
+      </listitem>
```

Comment:
Just add an alias to aliases.nix instead.

Reply:
It wasn't possible to add it because `lispPackages.nyxt` isn't a top-level package, so I made `lispPackages.nyxt` to throw an error message instead.